<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>937</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>794</width>
    <height>931</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IsTextRankKo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>501</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox{
border: none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Verhältnis:</string>
     </property>
    </widget>
    <widget class="QSlider" name="amountOfSent">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="leAmountOfSent">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Damping-Faktor:</string>
     </property>
    </widget>
    <widget class="QSlider" name="sDampingFactor">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDampingFactor">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>amountOfSent</zorder>
    <zorder>leAmountOfSent</zorder>
    <zorder>label_3</zorder>
    <zorder>label_6</zorder>
    <zorder>sDampingFactor</zorder>
    <zorder>leDampingFactor</zorder>
    <zorder>groupBox_3</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>791</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_2{
border: none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Geben Sie hier ihren Text ein:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>781</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startCal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Berechne</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>450</y>
      <width>801</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_3{
border: none;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ihr Ergebnis: </string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textOutput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>781</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>textOutput</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>groupBox_2</zorder>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>textEdit</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionOeffnen"/>
    <addaction name="actionErgebnis_exportieren_unter"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="action_ber_IsTextRankKo"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_ffnen">
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionSpeichern_unter">
   <property name="text">
    <string>Speichern unter</string>
   </property>
  </action>
  <action name="action_ffn">
   <property name="text">
    <string>Öffn</string>
   </property>
  </action>
  <action name="actionOeffnen">
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionErgebnis_exportieren">
   <property name="text">
    <string>Ergebnis exportieren</string>
   </property>
  </action>
  <action name="actionErgebnis_exportieren_unter">
   <property name="text">
    <string>Ergebnis speichern unter</string>
   </property>
  </action>
  <action name="action_ber_IsTextRankKo">
   <property name="text">
    <string>Über IsTextRankKo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
